articles/models.py

```diff
@@ -555,6 +590,9 @@ class SeriesArticleLink(Orderable, models.Model):
 class SeriesPage(Page, FeatureStyleFields, Promotable):
     subtitle = RichTextField(blank=True, default="")
     body = article_fields.BodyField(blank=True, default="")
+    cached_twitter_count = models.IntegerField(default=0)
```

Comment:
I'd filter the fields and the functions into a base class (add a class Meta with abstract = True)  that both articles and series inherit from, rather than duplicating the code.